src/docs/Chakra.mdx

```diff
@@ -346,111 +174,13 @@ const Button = {
   language="js"
 />
 
-### Colors
```

Comment:
Was this removed because it's redundant and appears in other files?

Reply:
Yeah. My thinking was that consumers care more about the tokens, then the underlying colors object and will be looking for token information in the `Colors` page rather than here.